README.adoc

```diff
@@ -277,9 +277,9 @@ Or just run the tests of a specific package (i.e. `cargo test -p srml-assets`)
 You can start a development chain with:
 
 [source, shell]
-cargo run \-- --dev
+cargo run --release -- --dev
 
-Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1 cargo run \-- --dev`.
+Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1 cargo run --release -- --dev`.
```

Comment:
```suggestion
Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1 cargo run --release \-- --dev`.
```

Reply:
what is the `\` for?

Reply:
That it is rendered as `--` and not as one `-`.